src/allencell_ml_segmenter/services/prediction_service.py

```diff
@@ -157,10 +157,8 @@ def build_overrides(self, checkpoint: Path) -> dict[str, Any]:
         channel: Optional[int] = (
             self._prediction_model.get_image_input_channel_index()
         )
-        if channel:
-            overrides["data.transforms.predict.transforms[1].reader[0].C"] = (
-                channel
-            )
+        if channel is not None:
```

Comment:
I feel like I always run into bugs like this--almost wish python didn't do 'truthy'/'falsy'

Reply:
js would shine here..